app/models/claim_update.rb

```diff
@@ -22,10 +22,15 @@ class ClaimUpdate
   }.freeze
 
   DEPENDENT_ELIGIBILITY_ANSWERS = {
-    "claim_school_id" => "employment_status",
     "had_leadership_position" => "mostly_performed_leadership_duties",
   }.freeze
 
+  DEPENDENT_EMPLOYMENT_ANSWERS = {}.freeze
+
+  ELIGIBILITY_ANSWERS_DEPENDENT_ON_EMPLOYMENT_ANSWERS = {
+    "school_id" => "employment_status",
```

Comment:
When we look at multiple schools, we should look at improving this to only reset it if they chose the school that's changed.

Reply:
Or maybe not at all? Just change the `employment_status` to be `:different_school`, and leave the current school alone?